### ISSUE TITLE
How to see loss per epoch when training?

### ISSUE THREAD

**Issue by Cabbagehust2507** [bgshih/aster#95]
Can you show me how to set up that?
I've just trained model with my own data. But I cannot see the loss per epoch  when training. It only indicated like this:
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:root:Number of classes is 94
INFO:root:UNK label is 2
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:root:Number of classes is 94
INFO:root:UNK label is 2
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Create CheckpointSaverHook.
2020-02-06 04:23:33.057670: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
INFO:tensorflow:Restoring parameters from aster/logs/model.ckpt-500
INFO:tensorflow:Saving checkpoints for 501 into aster/logs/model.ckpt.
INFO:tensorflow:global_step/sec: 2.27519
INFO:tensorflow:global_step/sec: 2.46377
INFO:tensorflow:global_step/sec: 2.46537
INFO:tensorflow:global_step/sec: 2.48628
INFO:tensorflow:Saving checkpoints for 1000 into aster/logs/model.ckpt.

**Comment by vaibhavnayak** [comment]
Were you able to see the loss? If so, could you please share how you did it. Thanks
